bin/pyrdp-player.py

```diff
@@ -21,36 +18,26 @@
 
 from PySide2.QtWidgets import QApplication
 
-from pyrdp.logging import LOGGER_NAMES, NotifyHandler
+from pyrdp.logging import LOGGER_NAMES, NotifyHandler, configure as configureLoggers
 from pyrdp.player import MainWindow
+from pyrdp.player.config import DEFAULTS
+from pyrdp.core import settings
 
-def prepareLoggers(logLevel: int, outDir: Path):
-    logDir = outDir / "logs"
-    logDir.mkdir(exist_ok = True)
-
-    textFormatter = logging.Formatter("[{asctime}] - {levelname} - {name} - {message}", style = "{")
-    notificationFormatter = logging.Formatter("[{asctime}] - {message}", style = "{")
-
-    streamHandler = logging.StreamHandler()
-    streamHandler.setFormatter(textFormatter)
-
-    fileHandler = logging.handlers.RotatingFileHandler(logDir / "player.log")
-    fileHandler.setFormatter(textFormatter)
+LOG = logging.getLogger(LOGGER_NAMES.PYRDP)
```

Comment:
We are using global loggers in programs under `bin/` and instance loggers in the core? Is that done consistently?

Reply:
All loggers are global when using the `logging` module, I'm not sure what you mean by instance logger

Reply:
I mean when the logger's lifetime follows one of an object.

Reply:
Since there are no objects for the binary scripts, it's a global logger, but there isn't much difference other than aesthetically  since the logging system is all global and `getLogger` only gives you a thread safe handle to the logger. The only place where it matters is for the connection objects which have custom formatters for their session ID. 

I'll do a quick pass in the code base to ensure that the style is consistent.